libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/BufferManagement/BufferAllocation_1.c

```diff
@@ -280,13 +280,14 @@ UBaseType_t uxCount;
 					pxReturn->pxNextBuffer = NULL;
 				}
 				#endif /* ipconfigUSE_LINKED_RX_MESSAGES */
-
+#if( ipconfigUSE_TCP_WIN != 0 )
```

Comment:
Could you show us the compiler issue log you have, without these directives surrounding, regarding this code and the one below?

Reply:
And are these issues you may be having here also no everywhere xTCPWindowLoggingLevel is located, for you?

Reply:
`In file included from amazon-freertos/libraries/freertos_plus/standard/freertos_plus_tcp/include/FreeRTOS_IP.h:34,
                 from amazon-freertos/libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/BufferManagement/BufferAllocation_1.c:44:
amazon-freertos/libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/BufferManagement/BufferAllocation_1.c: In function 'vReleaseNetworkBufferAndDescriptor':
./src/FreeRTOSIPConfig.h:60:9: error: macro expands to multiple statements [-Werror=multistatement-macros]
         while (xSemaphoreTake(xSemaphoreTrace,portMAX_DELAY) != pdPASS); \
         ^~~~~
./src/FreeRTOSIPConfig.h:50:41: note: in expansion of macro 'FreeRTOS_printf'
     #define FreeRTOS_debug_printf( X )  FreeRTOS_printf ( X )
                                         ^~~~~~~~~~~~~~~
amazon-freertos/libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/BufferManagement/BufferAllocation_1.c:394:4: note: in expansion of macro 'FreeRTOS_debug_printf'
    FreeRTOS_debug_printf( ( "BUF_PUT[%ld]: %p (%p) (now %lu)\n",
    ^~~~~~~~~~~~~~~~~~~~~
amazon-freertos/libraries/freertos_plus/standard/freertos_plus_tcp/source/portable/BufferManagement/BufferAllocation_1.c:393:3: note: some parts of macro expansion are not guarded by this 'if' clause
   if( xTCPWindowLoggingLevel > 3 )
   ^~
cc1: all warnings being treated as errors
Makefile:437: recipe for target 'obj/sram_BufferAllocation_1.o' failed
make: *** [obj/sram_BufferAllocation_1.o] Error 1`